tidb-lightning/tidb-lightning-backends.md

```diff
@@ -1,537 +1,119 @@
 ---
-title: TiDB Lightning Backends
-summary: Learn the backends of TiDB Lightning.
+title: TiDB Lightning Import Mode
+summary: Learn how to choose the different import modes of TiDB Lightning.
 aliases: ['/docs/dev/tidb-lightning/tidb-lightning-tidb-backend/','/docs/dev/reference/tools/tidb-lightning/tidb-backend/','/tidb/dev/tidb-lightning-tidb-backend','/docs/dev/loader-overview/','/docs/dev/reference/tools/loader/','/docs/dev/load-misuse-handling/','/docs/dev/reference/tools/error-case-handling/load-misuse-handling/','/tidb/dev/load-misuse-handling','/tidb/dev/loader-overview/']
 ---
 
-# TiDB Lightning Backends
+# TiDB Lightning Import Mode
 
-The backend determines how TiDB Lightning imports data into the target cluster.
+TiDB Lightning supports two import modes in two [backends](/tidb-lightning/tidb-lightning-glossary.md#back-end). The backend determines how TiDB Lightning imports data into the target cluster.
 
-TiDB Lightning supports the following [backends](/tidb-lightning/tidb-lightning-glossary.md#back-end):
+- The **Local-backend**: TiDB Lightning first encodes data into key-value pairs, sorts and stores them in a local temporary directory, and *upload* these key-value pairs to each TiKV node. Then, TiDB Lightning calls the TiKV ingest interface to write the data into RocksDB in TiKV. For initialized data import, consider the local-backend because it has a high import speed.
```

Comment:
```suggestion
- The **Local-backend**: TiDB Lightning first encodes data into key-value pairs, sorts and stores them in a local temporary directory, and *uploads* these key-value pairs to each TiKV node. Then, TiDB Lightning calls the TiKV ingest interface to write the data into RocksDB in TiKV. For initialized data import, consider the local-backend because it has a high import speed.
```